multimodal/tests/unittests/predictor/test_predictor.py

```diff
@@ -621,3 +632,39 @@ def test_modifying_duplicate_model_names():
     for per_modality_processors in teacher_predictor._data_processors.values():
         for per_processor in per_modality_processors:
             assert per_processor.prefix in teacher_predictor._config.model.names
+
+
+def test_image_bytearray():
+    dataset_1 = ALL_DATASETS["petfinder"]
```

Comment:
Can we add a vision-only dataset for unittest, like shopee? In this way, we don't need to specify other models names and hyperparameters.

Reply:
Good point, just updated.